content/source/docs/cloud/api/audit-trails.html.md

```diff
@@ -22,13 +22,11 @@ page_title: "Audit Trails - API Docs - Terraform Cloud"
 
 # Audit Trails API
 
--> **Note:** Audit trails are a paid feature, available as part of the **Terraform Cloud for Business** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing).
+Terraform Cloud retains 14 days of audit log information. The audit trails API exposes a stream of audit events, which describe changes to the application entities (workspaces, runs, etc.) that belong to a Terraform Cloud organization. Audit trails are a paid feature that is available as part of the **Terraform Cloud for Business** upgrade package. Refer to the [Terraform Cloud pricing page](https://www.hashicorp.com/products/terraform/pricing) for more details.
 
--> **Note:** Unlike other APIs, the Audit Trails API does not use the [JSON API specification](./index.html#json-api-formatting).
-
--> **Note:** Terraform Cloud retains 14 days of audit log information.
+-> **Note:** The Audit Trails API is not available for Terraform Enterprise.
 
-The audit trails API exposes a stream of audit events, which describe changes to the application entities (workspaces, runs, etc.) that belong to a Terraform Cloud organization.
+-> **Note:** Unlike other APIs, the Audit Trails API does not use the [JSON API specification](./index.html#json-api-formatting).
```

Comment:
my last thought is - I think this could just be a one-liner underneath the nice paragraph you h ave above :-) Then the only note here would be the TFE one. And if you do that, I approve! Thank you so much for doing this and for your patience with the editing process!

Reply:
ok, done!